src/File/FileHelper.php

```diff
@@ -31,6 +31,9 @@ public function absolutizePath(string $path): string
 				return $path;
 			}
 		}
+		if (\Nette\Utils\Strings::contains($path, '://')) {
```

Comment:
What about `startsWith()` and `phar://` only?

Reply:
works for me, I doubt anyone else would be using custom stream wrappers.